### DISCUSSION TITLE
[More Effective C#] Chapter 1. 요약

### DISCUSSION THREAD

**Discussion by hyun98** [BDD-CLUB/challenge-every-week-posting#370] [3. CS(컴퓨터 공학) 관련 글 모음]
안녕하세요! 처음 비글챌에 올리는 글이네요
업무에서 쓰는 C# 언어를 깊게 파보고자 More Effective C# 책을 공부해서 마음대로 정리해 보았습니다.
총 5개 챕터를 요약할 예정인데요 1챕터는 재미없는 파트라 좀 힘들었네요,,,
[[More Effective C#] Chapter 1. 요약](https://hyeo-noo.tistory.com/422)

**Comment by jojaeng2** [comment]
다시 블로그를 살려보아요
